core/lib/spree/app_configuration.rb

```diff
@@ -88,6 +88,10 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Boolean]
     preference :binary_inventory_cache, :boolean, default: false
 
+    # @!attribute [rw] build_shipments
+    #   @return [Boolean] should shipments be built (default: +true+)
+    preference :build_shipments, :boolean, default: true
```

Comment:
This seems a little weird as a preference because it isn't a behaviour we properly want to support. Maybe we could add some wording here to dissuade users from setting it to false.

Maybe:

> This setting is provided to help users who have made heavy modifications to disable shipments in their store. It should not be changed.